Lab_day-3_Dorota_Parina.js

```diff
@@ -0,0 +1,83 @@
+// Iteration 1: Names and Input
+
+var hacker1 = "Parina";
+console.log(`The driver's name is ` + hacker1);
+
+var hacker2 = "Dorota";
+console.log(`The navigator's name is ` + hacker2);
+
+
+// Iteration 2: Conditionals
+
+function theLongestName(name) {
+    if (hacker1.length > hacker2.length) {
+        return "The Driver has the longest name, it has " + hacker1.length + " characters"
+    } else if (hacker1.length < hacker2.length) {
+        return "Yo, navigator got the longest name, it has " + hacker2.length + " characters"
+    } else {
+        return "Wow, you both got equally long names " + hacker2.length + " characters!"
+    }
+}
+
+console.log(theLongestName());
+
+
+// Iteration 3: Loops
+
+function printToUppercase(str) {
+    var strUpperCase = '';
+    for (var i = 0; i < str.length; i++) {
+        strUpperCase += str[i].toUpperCase() + " ";
+    }
+    return strUpperCase;
+}
+console.log(printToUppercase(hacker1));
+
+
+
+
+
+
+function reverseString(str) {
+    let upperCasedFirstLetter = hacker2.slice(0, 1).toUpperCase();
+    console.log(upperCasedFirstLetter)
+
```

Comment:
your 2 first variable are useless... you are not calling them later.
But good job for the revers + join !